src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/CreateCaseCallbackService.java

```diff
@@ -100,30 +109,53 @@ public CreateCaseCallbackService(
         ExceptionRecord exceptionRecord,
         ServiceConfigItem configItem,
         long caseId,
-        boolean ignoreWarnings
+        boolean ignoreWarnings,
+        String idamToken,
+        String userId
     ) {
+        String caseIdAsString = Long.toString(caseId);
+
+        log.info("Start creating new case for {} from exception record {}", configItem.getService(), caseIdAsString);
+
         try {
-            log.info(
-                "Start creating exception record for service {} and exception record {}",
-                configItem.getService(),
-                caseId
-            );
+            String s2sToken = s2sTokenGenerator.generate();
 
             SuccessfulTransformationResponse transformationResponse = transformationClient.transformExceptionRecord(
                 configItem.getTransformationUrl(),
                 exceptionRecord,
-                s2sTokenGenerator.generate()
+                s2sToken
             );
 
             if (!ignoreWarnings && !transformationResponse.warnings.isEmpty()) {
-                log.warn("Transformation warnings: {}", String.join(", ", transformationResponse.warnings));
-
+                // do not log warnings
                 return Validation.invalid(Array.ofAll(transformationResponse.warnings));
```

Comment:
Maybe not necessarily for this PR, but this method needs to start returning a different kind of result. Currently, it returns:
* valid when successful response
* valid when errors
* invalid when warnings

Reply:
There's inconsistency I agree, but on the ccd it ends up in a same way - error'ed. There's a tiny possibility to achieve errors + warnings with this new design, but essentially it'll end up as errors disregardingly. The way it was before with another callback endpoint